code/game/machinery/computer/communications.dm

```diff
@@ -43,6 +43,9 @@
 /obj/machinery/computer/communications/New()
 	GLOB.shuttle_caller_list += src
 	..()
+
+/obj/machinery/computer/communications/Initialize(mapload)
```

Comment:
`GLOB.shuttle_caller_list` is surprisingly scary, so i'm not moving that to Initialize yet. Maybe next time.